scripts/library.sh

```diff
@@ -315,6 +315,51 @@ function check_licenses() {
   run_dep_collector -check $@
 }
 
+# Run gofmt to check format of .go files
+function check_format() {
```

Comment:
maybe we can create a prow plugin instead of hard-coding this for gofmt. Then we can /gofmt or /fmt on the PR and run it independently.

Reply:
k8s already has a script https://github.com/kubernetes/test-infra/blob/master/hack/verify-gofmt.sh that does gofmt. We can run that with Prow as well instead of creating a new plugin